bench/irmin-pack/trace_replay_intf.ml

```diff
@@ -13,19 +13,54 @@
  * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
-type config = {
-  ncommits_trace : int;
-  store_dir : string;
-  path_conversion : [ `None | `V1 | `V0_and_v1 | `V0 ];
-  inode_config : int * int;
-  store_type : [ `Pack | `Pack_layered | `Pack_mem ];
-  commit_data_file : string;
-  artefacts_dir : string;
-  keep_store : bool;
-  keep_stat_trace : bool;
-  no_summary : bool;
-  empty_blobs : bool;
-}
+
+module Config = struct
+  type config = {
+    number_of_commits_to_replay : int;
+    path_conversion : [ `None | `V1 | `V0_and_v1 | `V0 ];
```

Comment:
Shouldn't we use `version` instead of `path_conversion` here? Isn't it too close to the migration vocabulary?